controlplane/hack/deploy/addon/policy/manager/addon-deployment.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: grc
+    component: ocm-policy-addon-ctrl
+  name: grc-policy-addon-controller
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: grc
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: manager
+      labels:
+        app: grc
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: kubernetes.io/arch
+                    operator: In
+                    values:
+                      - amd64
+                      - ppc64le
+                      - s390x
+                      - arm64
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - podAffinityTerm:
+                labelSelector:
+                  matchExpressions:
+                    - key: ocm-antiaffinity-selector
+                      operator: In
+                      values:
+                        - grcpolicyaddon
+                topologyKey: topology.kubernetes.io/zone
+              weight: 70
+            - podAffinityTerm:
+                labelSelector:
+                  matchExpressions:
+                    - key: ocm-antiaffinity-selector
+                      operator: In
+                      values:
+                        - grcpolicyaddon
+                topologyKey: kubernetes.io/hostname
+              weight: 35
+      containers:
+        - args:
+            - controller
+          command:
+            - governance-policy-addon-controller
+            - "--controlplane"
+            - "--hubkubeconfig=/var/run/secrets/hub/kubeconfig"
+          env:
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: metadata.name
+            - name: CONFIG_POLICY_CONTROLLER_IMAGE
+              value: quay.io/open-cluster-management/config-policy-controller:latest
+            - name: KUBE_RBAC_PROXY_IMAGE
+              value: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.10
```

Comment:
need credential to pull this image?

Reply:
why need rbac proxy?

Reply:
the config-policy-controller agent pod will create proxy container when the managed cluster is OCP. 
https://github.com/open-cluster-management-io/governance-policy-addon-controller/blob/353c6ae127dc3b9b413438749b7f970f6559a8c1/pkg/addon/configpolicy/manifests/managedclusterchart/templates/deployment.yaml#L33